.github/workflows/sandpaper-main.yaml

```diff
@@ -32,7 +33,9 @@ jobs:
     steps:
 
       - name: "Checkout Lesson"
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
Presumably the version should be bumped here? (I copied the contents of https://github.com/swcarpentry-i18n/shell-novice/blob/main/.github/workflows/sandpaper-main.yaml)

Reply:
Presumably, yes. And the same probably goes for the other i18n repos, which do not seem to have been updated recently (e.g., https://github.com/swcarpentry-i18n/git-novice/blob/92a9370299427603e09189f71e620afe3daa38bc/.github/workflows/sandpaper-main.yaml#L36)